ui/packages/catalog-ui-search/src/main/webapp/js/model/QueryResponse.js

```diff
@@ -47,6 +47,18 @@ function humanizeResourceSize(result) {
     }
 }
 
+function handleResultFormFields(result, selectedResultTemplate) {
+    if (selectedResultTemplate) {
+        var resultAttributes = selectedResultTemplate.descriptors;
```

Comment:
All new files should use ES6 syntax, as well as `require` instead of `define`.
Existing files are developer's choice, and keeping style consistent in the file is totally fine.

Reply:
The @andrewkfiedler has spoken!